src/style.css

```diff
@@ -1 +1,54 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link a {
```

Comment:
please try to avoid using simple tags in your rules
it is better add to element you want special class
because now it will affects all links that you will added in the next work